### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by stumblefiend**
Increases readability by 7% based on Flesch-Kincaid readability scores and decreases time-to-read by 22%. Especially handy for non-native English speakers.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,23 +61,23 @@ rye run lint
...
-We have samples in the `examples` directory and using them for testing can help increase your productivity.
+Samples are in the `examples` directory. Using samples for testing can increase your productivity.
 
-Make sure your notebook will use your source code:
+Make sure your notebook uses your source code:
 1. `ipython profile create`
-2. `ipython locate` - will show where the config file is. This is the config file used by the jupyter server, since it runs via ipython
-3. Open the file (this should be ipython_config.py and it is usually located in ~/.ipython/profile_default) and add the following line of code: 
+2. `ipython locate` - shows the location of the jupyter server config file.
+3. Open the ipython_config.py file (usually located in ~/.ipython/profile_default/) and add the following line of code:
```
Comment (stumblefiend) [inline CONTRIBUTING.md]:
```suggestion
3. Open the ipython_config.py file (usually located in ~/.ipython/profile_default/) and add the following lines of code: 
```

**Review by tomislav-peharda — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,45 +16,42 @@ This is a general guide applicable to all modules in the repository.
 
 ## General Guidelines
 
-Thank you for expressing your interest in contributing! To ensure that your contribution aligns with our guidelines, please carefully review the following before proceeding:
+Thank you for your interest in contributing! To ensure that your contribution aligns with our guidelines, review the following tips:
```
Comment (stumblefiend) [inline CONTRIBUTING.md]:
```suggestion
Thank you for your interest in contributing! To ensure that your contribution meets our guidelines, review the following tips:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,45 +16,42 @@ This is a general guide applicable to all modules in the repository.
...
+* For feature requests, ask [Labelbox Support](https://docs.labelbox.com/docs/contacting-customer-support) or create a [Github Issue](https://github.com/Labelbox/labelbox-python/issues).
+* We accept general solutions that address common issues rather than specific use cases. Example contributions include a wide range of activities, such as bug fixes and updates to dependencies.
+* Ensure that any new libraries comply with this repository's Apache license.
+* Update any relevant docstrings and comments in the code you add or change.
+* Ensure that new Python components, such as classes, packages, or methods, that need to feature in the Labelbox documentation have entries in the file [index.rst](https://github.com/Labelbox/labelbox-python/blob/develop/docs/source/index.rst).
+* Add entries in [index.rst](https://github.com/Labelbox/labelbox-python/blob/develop/docs/source/index.rst) for new Python components, such as classes, packages, or methods, that need to feature in the Labelbox documentation.
```
Comment (mrobers1982) [inline CONTRIBUTING.md:26]: This is essentially a duplicate of the line above.

**Comment by adrian-chang** [comment]
I think we can merge this without tests given it's a readme update

File: CONTRIBUTING.md (hunk 3 above)
Comment (adrian-chang) [reply CONTRIBUTING.md:26]: Please fix and we can add your contribution. Thank you!

**Comment by adrian-chang** [comment]
Closing per lack of activity ... please reopen if needed

**Pull request closed without merging**